### ISSUE TITLE
[Bug Report][3.8.0] v-select opens on mouse-down, violating WCAG 2.5.2 Input modalities - Pointer cancellation

### ISSUE THREAD

**Issue by hellmanhellman** [vuetifyjs/vuetify#21192]
### Environment
**Vuetify Version:** 3.8.0
**Vue Version:** 3.5.13
**Browsers:** Edge 134.0.0.0, Google Chrome, Mozilla Firefox
**OS:** Mac OS 10.15.7

### Steps to reproduce
Click a v-select.

### Expected Behavior
The menu opens on mouse-down, which violates WCAG 2.1:

https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html

### Actual Behavior
The menu should open on mouse-up.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVkMGqwjAQRX/lkk3fg/q6D32CG934B9ZFaEcJjG1IJ0Uo/XdjirYu3SRzT+6dDHMaVe/rYufc3xBIaVUK3RwboW3VAuWw6YmpFgybW9cQ/1dqBpWCttHbR5LuCIqYKYtVgyj72lsnqRndXecFDV1MYMH4ZEBjxPz8vhTgSYJvFw3MH2qMECtMGtmeO28bk+UYDIdEjhmmfMmkkTROC8FX+bX7QJ2/2rX7sPt0n9/1NBfzYxRl8V6AmvI2MKfj/ADm9nKC)

**Comment by hellmanhellman** [comment]
Note: If you want to preserve the "open on mousedown" behavior, perhaps you could add a prop that allows us change that behavior so that we can comply with WCAG if we want to.
